scripts/check_file_license_apache.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+FLAG=$1
+FILE=$2
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+TIMESCALE_LOCATION=`dirname $SCRIPTPATH`
+
+LICENSE_STRING=
+FIRST_COMMENT=
+
+if [[ "$FLAG" == '-c' ]]; then
+    LICENSE_STRING='/*  * Copyright (c) 2018  Timescale, Inc. All Rights Reserved.  *  * This file is licensed under the Apache License,  * see LICENSE-APACHE at the top level directory.  */ '
+    FIRST_COMMENT=`awk 'BEGIN {ORS=" "}{if($1 == "*/") {print; exit;}} {print}' $FILE`
+elif [[ "$FLAG" == '-s' ]]; then
+    LICENSE_STRING='-- Copyright (c) 2018  Timescale, Inc. All Rights Reserved. -- -- This file is licensed under the Apache License, see LICENSE-APACHE -- at the top level directory of the timescaledb distribution.  '
+    FIRST_COMMENT=`awk 'BEGIN {ORS=" "}{if($1 == "") {print; exit;}} {print}' $FILE`
+elif [[ "$FLAG" == '-t' ]]; then
```

Comment:
I don't see a need for the `-t` flag. Can you just make the license string for the `-t` case equal to the license string for the `-s` case (and similarly in the actual files)?

Reply:
They are slightly different atm, since the license on the other SQL files needs to make sense when read from the install directory.